### PR TITLE
feat(gen): Show version in console, add to .yo-rc.json

### PR DESCRIPTION
**Pull request by Awk34**

### CODE REVIEW

File: app/generator.js [hunk 1]
```diff
@@ -30,13 +31,16 @@ export default class Generator extends Base {
     return {
 
       init: function () {
-        this.filters = {};
+        this.filters = {
+          generatorVersion: version
```
Comment (kingcody) [inline app/generator.js]: We can use `this.rootGeneratorVersion()` instead of `version`.

File: app/generator.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import path from 'path';
 import chalk from 'chalk';
 import {Base} from 'yeoman-generator';
 import {genBase} from '../generator-base';
+import {version} from '../package.json';
```
Comment (kingcody) [inline app/generator.js]: Then we don't need this line.

**Comment by kingcody** [comment]
And a rebase.

**Comment by Awk34** [comment]
done

**Comment by kingcody** [comment]
Thanks

**Comment by kingcody** [comment]
One question, why not add it as a top level property, instead of putting it in filters?

**Comment by kingcody** [comment]
`this.config.set('generatorVersion', this.rootGeneratorVersion())`

**Comment by Awk34** [comment]
..oversight, I'll change

**Pull request merged**